redwood-layout-dom/src/commonMain/kotlin/app/cash/redwood/layout/dom/HTMLFlexElementChildren.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.layout.dom
+
+import app.cash.redwood.layout.modifier.Margin
+import app.cash.redwood.widget.HTMLElementChildren
+import app.cash.redwood.widget.Widget
+import org.w3c.dom.HTMLElement
+
+public class HTMLFlexElementChildren(
```

Comment:
internal

Might have to specify the type of the override in the other class to avoid exposing this type.